### ISSUE TITLE
SELKS login crash after a few days of use

### ISSUE THREAD

**Issue by Koploseus** [StamusNetworks/SELKS#212]
Hello,

I am using SELKS since 2-3 weeks now, since today i can't access (or randomly) to the Web interface, i get an error login message (even in private mode)

![image](https://user-images.githubusercontent.com/25018500/75519334-4529f200-5a03-11ea-9469-5fed2ab81575.png)

I can log in .. but when i select Evebox or Kibana..its failling

**Comment by pevma** [comment]
Hi,

Did you adjust the SELKS name in the /etc/scirius/local_settings.py ?
(You would need to restart Scirius after that)

Thank you 

> -- 
> Regards,
> Peter Manev

**Comment by Koploseus** [comment]
Yes already, but keep crashing.. and now its crashing the physical machine

**Comment by Koploseus** [comment]
I have re installed everything.. so i will see this week if its happening again

**Comment by pevma** [comment]
Can you please share when it fails - how many events etc do you have in and what is the HW/VM setup you have - CPUs/RAM?
Also when it happen if you could share the complete output of `selks-healthcheck_stamus`.

**Comment by Koploseus** [comment]
4 CPU and 12G of RAM.
I can't see the selks-health because i can't access to the machine anymore :) 

I had to re install all the stuff ! :)

**Comment by pevma** [comment]
I think what may happen is that the box could be under powered for the size of traffic you are monitoring  - could that be the case?

**Comment by Koploseus** [comment]
I thought about that, but i didn't have any problems before, its crashing only after an update

**Comment by pevma** [comment]
thats - `selks-upgrade_stamus` ? 
Were there any errs during upgrade?
